### DISCUSSION TITLE
Home Assistant Theme related LXC that was on tteck?

### DISCUSSION THREAD

**Discussion by gravelfreeman** [community-scripts/ProxmoxVE#350] [Q&A]
I'm relatively new with Proxmox but I remember going through the tteck website and taking notes about some stuff that I was interested in. If I remember correctly what I'm searching for, it was close to the homeassistant LXC/VM on his website. It was an add-on with full blown theming/dashboard stuff. It looked like iOS but it wasn't the iOS theme.

I wanted to try it but it seems like I can't find it anymore.

Thanks for your help!

**Comment by Node815** [comment]
I've used the image from the old site for a couple of years now, and there was no theme which was added through the install.  I believe the default process is/was to create a default install and let the end user decide what to install and customize after. 

As for Home Assistant themes, you can look into installing the HACS addon which will open up a whole new world of addons for Home Assistant and contains many themes and other goodies.

**Comment by MickLesk** [comment]
Never heard this before. And im a while active in this Project 😅

**Comment by gravelfreeman** [comment]
Well I might be mistaken hehe! Sorry for the confusion.
